README.md

```diff
@@ -346,13 +347,30 @@ Builders are unnecessary, and are not included by default.  For compatibility wi
 
 By default, Thrifty generates one Kotlin file per JVM package.  For larger thrift files, this can be a little hard on the Kotlin compiler.  If you find build times or IDE performance suffering, the `--kt-file-per-type` flag can help.  Outlook Mobile's single, large, Kotlin file took up to one minute just to typecheck, using Kotlin 1.2.51!  For these cases, `--kt-file-per-type` will tell Thrifty to generate one single file per top-level class - just like the Java code.
 
+#### Server Support
+Support for generating a server implementation was only added very recently, and while it passes the 'official'
+[Java client integration test](https://github.com/apache/thrift/blob/master/lib/java/test/org/apache/thrift/test/TestClient.java),
+you should consider this code experimental.
+
+Thrifty generates a `Processor` implementation that you pass an input `Protocol`, an output `Protocol` and a service handler
+and the code will take care of reading the request, passing it to the handler and returning the correct response to the output.
+
+If you want to use it, you need to wrap an appropriate communication layer around it, e.g. an HTTP server.
+You can have a look at the [integration tests](thrifty-integration-tests/src/test/kotlin/com/microsoft/thrifty/integration/conformance/server/TestServer.kt) for a basic example.
+
 ### Java-specific command-line options
 
 Thrifty can be made to add various kinds of nullability annotations to Java types with the `--nullability-annotation-type` flag.  Valid options are
 `none` (the default), `android-support`, and `androidx`.  Specifying `android-support` will cause generated code to use `@Nullable` and `@NonNull` from
 the `android.support.annotation` package.  Similarly, specifying `androidx` will use analogous annotations from `androidx.annotation`. 
 
 ## Thanks
+`--experimental-kt-generate-server` enabled code generation for the server portion of a thrift service. You can
+use this to implement a thrift server with the same benefits as the kotlin client: no runtime surprises thanks to
+structs being always valid by having nullability guarantees and unions represented as sealed classes.
+See 
+
+### Thanks
```

Comment:
This readme block looks misplaced.  Is this an artifact of the rebase?

Reply:
yes, seems I somehow messed that up. Fixed now.